compiler/ballerina-lang/src/main/resources/grammar/BallerinaParser.g4

```diff
@@ -19,7 +19,13 @@ packageName
     ;
 
 version
-    :   VERSION Identifier
+    :   VERSION versionPattern
```

Comment:
One possible improvement we can do is start a different lexer mode, with VERSION token and reset it with a semi-colon.